packages/core/useCycleList/index.ts

```diff
@@ -1,4 +1,4 @@
-import { type Ref, computed, shallowRef } from 'vue-demi'
+import { type Ref, computed, ref, shallowRef } from 'vue-demi'
```

Comment:
It is better to import the type type separately.

Reply:
Thank you. I have updated PR. 😁